### ISSUE TITLE
Issue with comment nesting on activity page when activity is posted with media

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1646]
There seems to be an issue with comment nesting on the activity page. Comment nesting structure looks broken. Once the page is refreshed the nesting structure does not maintain! 

Please check this visual after the page is refreshed - https://drive.google.com/file/d/19BR3UbyBUh3-XivXlmdO0lx1EX7fzNIz/view

Before the page is refreshed (expected) -
![Screen Shot 2020-05-05 at 9 46 39 AM](https://user-images.githubusercontent.com/15173772/81034680-90ee9100-8eb5-11ea-9954-79132544911a.jpg)

-----

```
Test Environment :

WP - V5.4.1
rtMedia - V4.6.2, V4.6.3
BuddyPress - V5.2.0
```

`2576463 `

**Comment by SID177** [comment]
The comment were intentionally implemented as non-threaded. The code was written by **Ritesh Patel** and that too in 2014.

Removing this code works but need to find why it was added in the 1st place.

Code - https://github.com/rtCamp/rtMedia/blob/master/app/main/controllers/activity/RTMediaBuddyPressActivity.php#L38

**Comment by pooja-muchandikar** [comment]
@SID177 

Cross-checked this issue with rtMedia previous releases and seems like it was working before. 

Can you please check v4.5.8 once?

**Comment by pooja-muchandikar** [comment]
@SID177 

I have tested the nested comments on profile activity page and group activity page and works fine. The comments looks like this
![image](https://user-images.githubusercontent.com/41000648/86902346-7641ce00-c12b-11ea-8750-05d80761eef6.png)


```
Test Environment
rtMedia: v4.6.3 pre-release branch
BuddyPress: v6.0.0 Nouveau and Legacy template
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
